framework/packages/abstract-app/src/endpoints/instantiate.rs

```diff
@@ -54,6 +54,9 @@ impl<
             address: deps.api.addr_validate(&version_control_address)?,
         };
 
+        // TODO: Would be nice to remove context
+        // Issue: We can't pass easily AccountBase with BaseInstantiateMsg(right now)
```

Comment:
https://linear.app/abstract-sdk/issue/ABS-229/remove-requirement-for-base-message-in-app-instantiate-message